src/Sylius/Bundle/ShopBundle/Provider/DefaultPayResponseProvider.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ShopBundle\Provider;
+
+use Sylius\Bundle\ResourceBundle\Controller\RequestConfiguration;
+use Sylius\Component\Core\Model\OrderInterface;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\RouterInterface;
+
+final class DefaultPayResponseProvider implements PayResponseProviderInterface
+{
+    public function __construct(
+        private RouterInterface $router,
+    ) {
+    }
+
+    public function getResponse(
+        RequestConfiguration $requestConfiguration,
+        OrderInterface $order
+    ): Response {
+        $url = $this->router->generate('sylius_shop_order_thank_you');
+
+        return new RedirectResponse($url);
+    }
+    public function supports(
```

Comment:
Missing empty line between methods.

```suggestion
    }
    
    public function supports(
```